gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -174,7 +174,6 @@
         </module>
         <module name="MethodParamPad"/> <!-- Java Style Guide: Horizontal whitespace -->
         <module name="MissingDeprecated"/> <!-- Java Coding Guide: Deprecate per annotation and Javadoc -->
-        <module name="MissingSwitchDefault"/> <!-- Java Style Guide: The default case is present -->
```

Comment:
also need to update docs: https://github.com/palantir/gradle-baseline/tree/develop/docs/java-style-guide#the-default-case-is-present